tests/Console/Command/DescribeCommandTest.php

```diff
@@ -32,18 +34,31 @@
  */
 final class DescribeCommandTest extends TestCase
 {
+    /**
+     * @var Application
+     */
+    private $application;
+
+    protected function setUp()
+    {
+        $this->application = new Application();
```

Comment:
I don't see much value in creating the application in `setUp()` so I would keep doing it in each test instead. But if you prefer this way, then please clear the property in `tearDown()` to:
- prevent side effects between tests;
- free memory after all tests;